chain/epoch_manager/src/tests/random_epochs.rs

```diff
@@ -1,418 +0,0 @@
-use rand::prelude::StdRng;
-use rand::{Rng, SeedableRng};
-use std::collections::{BTreeMap, BTreeSet, HashMap};
-
-use crate::test_utils::{
-    hash_range, record_block_with_slashes, setup_default_epoch_manager, stake,
-};
-use crate::EpochManager;
-use near_primitives::challenge::SlashedValidator;
-use near_primitives::epoch_manager::block_info::BlockInfo;
-use near_primitives::epoch_manager::epoch_info::EpochInfo;
-use near_primitives::epoch_manager::SlashState;
-use near_primitives::hash::CryptoHash;
-use near_primitives::types::validator_stake::ValidatorStake;
-use near_primitives::types::{AccountId, Balance, EpochId, ValidatorKickoutReason};
-
-const DEBUG_PRINT: bool = false;
-
-#[test]
```

Comment:
This seems to be a real test we are deleting, no?

Reply:
Yep, are right. Was a little bit confused since `cargo check` showed me that there was an unused code. Placed all that staff under a tests module.